### ISSUE TITLE
Remove Google analytics cookies

### ISSUE THREAD

**Issue by jdanish** [netcreateorg/netcreate-itest#226]
Noticed this warning recently, which appears to be tied to google analytics. Can we just strip out the google analytics code? I thought it was being ignored, but not completely? 

<img width="1717" alt="Screenshot 2024-05-30 at 7 26 37 PM" src="https://github.com/netcreateorg/netcreate-itest/assets/5529898/ef785f0c-33ce-410d-b2ab-f602ceb1bcce">

**Comment by benloh** [comment]
Just to make sure I understand: You want to remove ALL google analytics calls?  In other words you no longer want to use GA anywhere?  I know we had some embedded in the startup scripts and elsewhere.

**Comment by jdanish** [comment]
That’s correct. This is not urgent but we haven’t used this in several years so might as well clean it out.
